.eslintrc.json

```diff
@@ -1,38 +1,41 @@
 {
-    "env": {
-        "node": true
-    },
-    "extends": "eslint:recommended",
-    "rules": {
-        "comma-spacing": ["error", {
-            "before": false,
-            "after": true
-        }],
-        "indent": [
-            "error",
-            2
-        ],
-        "key-spacing": ["error", {
-          "mode": "strict"
-        }],
-        "linebreak-style": [
-            "error",
-            "unix"
-        ],
-        "no-multi-spaces": "error",
-        "quotes": [
-            "error",
-            "single"
-        ],
-        "semi": [
-            "error",
-            "always"
-        ],
-        "space-in-parens": ["error", "never"],
-        "space-before-blocks": "error",
-        "space-before-function-paren": ["error", {
-            "anonymous": "always",
-            "named": "never"
-        }]
-    }
+  "env": {
+    "node": true
+  },
+  "extends": "eslint:recommended",
+  "plugins": [
+    "json"
+  ],
```

Comment:
If using the `json` plugin, should also add `npm install --save-dev eslint-plugin-json`